spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -88,20 +88,21 @@ class HyperConformance (implicit cmd: ConformanceConfig,
 /**
  * This is the definition of Dynamic Conformance as a component of Hyperdrive.
  *
- * In order to use it in hyperdrive the component needs to be configured in 'ingestion.properties' as follows:
+ * See https://github.com/AbsaOSS/hyperdrive#configuration for instructions how the component needs to be configured in Hyperdrive.
+ * Example values are given below:
  * {{{
- * transformer.hyperconformance.menas.rest.uri=http://localhost:8080
- * transformer.hyperconformance.dataset.name=example
- * transformer.hyperconformance.dataset.version=1
- * transformer.hyperconformance.report.date=2020-01-29
- * transformer.hyperconformance.report.version=1
- * transformer.hyperconformance.event.timestamp.column=EV_TIME
+ * menas.rest.uri=http://localhost:8080
```

Comment:
`menas.rest.uri` is already being used as a property key. Are we not scared of theses merging and overwriting themselves?

Reply:
The properties listed here have to be passed to Hyperdrive, not Enceladus. In the Hyperdrive config, there will not be a conflict.

More detail about Hyperdrive config: In Hyperdrive, the user has to prefix the properties with a prefix that he chooses. Hyperdrive then passes only the properties with that prefix to `HyperConformance`. Because the prefix is not static, it will be stripped off, before passing to the HyperConformance Transformer